src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/xlf/tr/BuildPropertyPage.xaml.xlf.lcl

```diff
@@ -12,19 +12,25 @@
       <Disp Icon="Str" Disp="true" LocTbl="false" />
       <Item ItemId=";BoolProperty|AllowUnsafeBlocks|Description" ItemType="0" PsrId="308" Leaf="true">
         <Str Cat="Text">
-          <Val><![CDATA[Allows code that uses the 'unsafe' keyword to compile.]]></Val>
+          <Val><![CDATA[Allow code that uses the 'unsafe' keyword to compile.]]></Val>
           <Tgt Cat="Text" Stat="Loc" Orig="New">
-            <Val><![CDATA['unsafe' anahtar sözcüğünü kullanan kodun derlenmesine izin verir.]]></Val>
+            <Val><![CDATA['güvenli olmayan' anahtar sözcüğü kullanan kodun derlenmesine izin verin.]]></Val>
```

Comment:
`unsafe` here is a keyword and should not be translated.

Reply:
I've left this kinda sitting here because we don't have a system to send feedback to translators. So, we'll just keep getting PRs that change stuff we don't want changed. I have no idea what system was in place prior, or if there is a system for that now.

Reply:
The issue with not accepting the translations as-is... I don't know how they handle new things that need translated if we don't merge the previous things. I don't have knowledge of the their system... I only connected to it from our repo.

Reply:
Previously these PRs were submitted by someone from the translation team, and feedback here was addressed. It makes sense that we review these translations, as we know the context behind them, even if we don't know the target languages.

Do you have a contact on the localisation team we can discuss this with?

Reply:
@csigs are you the right person to talk to here?

Reply:
I'm trying to get a contact. I sent a message to @cristianosuzuki77 to see if we can get some information on the process.